tests/src/tracing/common/NetperfFile.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace Tracing.Tests.Common
+{
+    public class NetperfFile : IDisposable
+    {
+        public string Path { get; }
+        private bool keepOutput { get; }
```

Comment:
`keepOutput` => `KeepOutput`. Or make it a private readonly field named `_keepOutput`?